x/stake/queryable.go

```diff
@@ -0,0 +1,206 @@
+package stake
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/wire"
+	keep "github.com/cosmos/cosmos-sdk/x/stake/keeper"
+	"github.com/cosmos/cosmos-sdk/x/stake/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// TODO Redelegations
+func NewQuerier(k keep.Keeper) sdk.Querier {
+	return func(ctx sdk.Context, path []string, req abci.RequestQuery) (res []byte, err sdk.Error) {
+		switch path[0] {
+		case "validators":
```

Comment:
you mean using `const` ? or [iota](https://golang.org/ref/spec#Iota) ?

Reply:
iota/enum, yeah 👍

Reply:
I think I prefer to go for `const` instead since it's what we've been using on the sdk. Let's keep the consistency as it's been suggested previously. Otherwise we can submit a proposal to the team

Reply:
@fedekunze we actually use both. I think constants here would suffice as well 👍